include/robot_interfaces/robot_logger.hpp

```diff
@@ -124,74 +115,78 @@ class RobotLogger
             desired_action.get_data();
         std::vector<std::vector<double>> status_data = status.get_data();
 
-        header_.push_back("#");
-        header_.push_back("[Timestamp]");
-        header_.push_back("Time Index");
+        std::vector<std::string> header;
 
-        append_name_to_header("(S)", status_name, status_data);
-        append_name_to_header("(O)", observation_name, observation_data);
-        append_name_to_header("(AA)", applied_action_name, applied_action_data);
-        append_name_to_header("(DA)", desired_action_name, desired_action_data);
+        header.push_back("#");
+        header.push_back("[Timestamp]");
+        header.push_back("Time_Index");
 
-        return header_;
+        append_name_to_header("(S)", status_name, status_data, header);
+        append_name_to_header(
+            "(O)", observation_name, observation_data, header);
+        append_name_to_header(
+            "(AA)", applied_action_name, applied_action_data, header);
+        append_name_to_header(
+            "(DA)", desired_action_name, desired_action_data, header);
 
+        return header;
     }
 
-    /*
-    * append_name_to_header() fills in the name information of each field to be logged
-    * according to the size of the field.
-    */
-
-    void append_name_to_header(std::string identifier, std::vector<std::string> field_name,
-                               std::vector<std::vector<double>> field_data)
+    /**
+     * @brief Fills in the name information of each field to be
+     * logged according to the size of the field.
+     *
+     * @param The structure the field corresponds to
```

Comment:
The doxygen [`@param`](http://www.doxygen.nl/manual/commands.html#cmdparam) tag expects the name of the parameter as first argument, i.e. it should be

    @param  identifier  The structure...

Reply:
Sorry :|, fixed.